### PR TITLE
[IMP] l10n_au: update account template

### PR DESCRIPTION
**Pull request by tong-odoo**
Impacted Version:

- 16.4 and above

This commit improve below features:

- Set default production account
- Update some taxes tax computation
- Rename "A/P Accruals" account
- Update some taxes description
- Translate label "Tax" to "GST"

task-3419980

### CODE REVIEW

File: addons/l10n_au/data/template/account.tax-au.csv [hunk 1]
```diff
@@ -54,20 +54,20 @@
...
 "","","","","","","","","","","","","base","refund","","-G11||-G14"
 "","","","","","","","","","","","","tax","refund","","l10n_au.service_tag"
-"au_tax_purchase_0_service_tpar_no_abn","0% C TPAR N ABN","4","Capital Purchases","Capital Purchases","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_0_service_tpar,au_tax_witheld","","","",""
-"au_tax_purchase_taxable_import_service","100% T EX","5","Purchase (Taxable Imports) - Tax Paid Separately","Purchase (Taxable Imports) - Tax Paid Separately","purchase","","percent","100.0","True","tax_group_gst_100000000","","base","invoice","","+G11||+G14"
+"au_tax_purchase_0_service_tpar_no_abn","0% C TPAR N ABN","4","Zero Rated Purch TPAR without ABN","Zero Rated Purch TPAR without ABN","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_0_service_tpar,au_tax_witheld","","","",""
+"au_tax_purchase_taxable_import_service","100% T EX","5","Purchase (Taxable Imports) - Tax Paid Separately","Purchase (Taxable Imports) - Tax Paid Separately","purchase","","division","100.0","True","tax_group_gst_100000000","","base","invoice","","+G11||+G14"
```
Comment (vin-odoo) [inline addons/l10n_au/data/template/account.tax-au.csv:66]: The only place in the task that this is talked about says to not change anything.
May I know where this change comes from?

File: addons/l10n_au/data/template/account.tax-au.csv [hunk 2]
```diff
@@ -54,20 +54,20 @@
...
 "","","","","","","","","","","","","tax","invoice","",""
 "","","","","","","","","","","","","base","refund","","-G11||-G14"
 "","","","","","","","","","","","","tax","refund","",""
-"au_tax_purchase_taxable_import_service_tpar","100% T EX TPAR","5","Purchase (Taxable Imports) - Tax Paid Separately","Purchase (Taxable Imports) - Tax Paid Separately","purchase","service","percent","100.0","True","tax_group_gst_100000000","","base","invoice","","+G11||+G14"
+"au_tax_purchase_taxable_import_service_tpar","100% T EX TPAR","5","Purchase (Taxable Imports) - Tax Paid Separately","Purchase (Taxable Imports) - Tax Paid Separately","purchase","service","division","100.0","True","tax_group_gst_100000000","","base","invoice","","+G11||+G14"
```
Comment (vin-odoo) [inline addons/l10n_au/data/template/account.tax-au.csv:70]: Same thing

File: addons/l10n_au/data/template/account.tax-au.csv (hunk 1 above)
Comment (tong-odoo) [reply addons/l10n_au/data/template/account.tax-au.csv:66]: They want to change it to this one at first but in the end still wrong. So they will keep looking for the correct one

Comment (tong-odoo) [reply addons/l10n_au/data/template/account.tax-au.csv:66]: @vin-odoo Should we go ahead first or remove the update in this case?

Comment (vin-odoo) [reply addons/l10n_au/data/template/account.tax-au.csv:66]: Better to not update something wrong by another wrong thing, at least it won't confuse peoples as to why it was updated this way

Comment (tong-odoo) [reply addons/l10n_au/data/template/account.tax-au.csv]: Updated

**Review by vin-odoo — APPROVED** [review]

**Comment by vin-odoo** [comment]
@robodoo r+

**Comment by vin-odoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**